### PR TITLE
feat: add track picking and truth seed selection

### PR DESCRIPTION
**Pull request by asalzburger**
This PR addresses (at least partly)  #775.

It adds:
 * allow for single track picking in TruthTrackFitting for screen debugging
 * feed the TruthSeedSelector with program options

### CODE REVIEW

File: Examples/Algorithms/TruthTracking/ActsExamples/TruthTracking/TruthSeedSelector.cpp [hunk 1]
```diff
@@ -8,18 +8,68 @@
...
+      return;
+    }
+    auto interval = vars[name].as<Options::Interval>();
+    lower = interval.lower.value_or(lower) * unit;
+    upper = interval.upper.value_or(upper) * unit;
+  };
+
+  Config cfg;
+  extractInterval("select-rho-mm", 1_mm, cfg.rhoMin, cfg.rhoMax);
+  extractInterval("select-absz-mm", 1_mm, cfg.zMin, cfg.zMax);
```
Comment (XiaocongAi) [inline Examples/Algorithms/TruthTracking/ActsExamples/TruthTracking/TruthSeedSelector.cpp:64]: I guess it should be `select-z-mm`?

**Review by XiaocongAi — APPROVED** [review]
Hi @asalzburger , it looks all good except one naming you might want to fix.

File: Examples/Algorithms/TruthTracking/ActsExamples/TruthTracking/TruthSeedSelector.cpp (hunk 1 above)
Comment (asalzburger) [reply Examples/Algorithms/TruthTracking/ActsExamples/TruthTracking/TruthSeedSelector.cpp]: Fixed.

**Comment by asalzburger** [comment]
This has been approved by @XiaocongAi already, we lost the approval via the `main` renaming.

**Pull request merged**